src/GitHub/Request.hs

```diff
@@ -256,8 +256,8 @@ makeHttpRequest auth r = case r of
     setAuthRequest _                                  = id
 
     getOAuthHeader :: Auth -> RequestHeaders
-    getOAuthHeader (OAuth token) = [("Authorization", BS8.pack ("token " ++ token))]
-    getOAuthHeader _                   = []
+    getOAuthHeader (OAuth token) = [("Authorization", "token " <> token)]
+    getOAuthHeader _             = []
```

Comment:
Not sure if that's a problem, but shouldn't it add the token in case of `EnterpriseOAuth`?